### PR TITLE
Projectkk2glider/misc fixes

### PR DESCRIPTION
**Pull request by projectkk2glider**
Partial fixes for #4326 and #4329

### CODE REVIEW

File: companion/src/firmwares/opentx/opentxinterface.cpp [hunk 1]
```diff
@@ -155,7 +155,8 @@ bool OpenTxEepromInterface::saveToByteArray(const T & src, QByteArray & data, ui
     version = getLastDataVersion(getBoard());
   }
   QByteArray raw;
-  M manager((T&)src, board, version, 0);
+  T srcCopy(src); // work on a copy of radio data, because Export() will modify it!
```
Comment (mpaperno) [inline companion/src/firmwares/opentx/opentxinterface.cpp]: Isn't `src` already `const` here?

Comment (projectkk2glider) [inline companion/src/firmwares/opentx/opentxinterface.cpp]: Notice the sneaky cast.

Comment (mpaperno) [inline companion/src/firmwares/opentx/opentxinterface.cpp]: Yikes.

File: companion/src/eeprominterface.cpp [hunk 2]
```diff
@@ -1075,6 +1075,9 @@ GeneralSettings::GeneralSettings()
     switchConfig[i] = getSwitchInfo(board, i).config;
   }
 
+  backlightMode = 3; // keys and sticks
```
Comment (bsongis) [inline companion/src/eeprominterface.cpp]: Ouch we will need some constant soon ;)

File: companion/src/firmwares/opentx/opentxinterface.cpp (hunk 1 above)
Comment (bsongis) [inline companion/src/firmwares/opentx/opentxinterface.cpp]: ouch yes, was done quick & dirty by me!

**Pull request merged**